watch_faces/BinarySystem/source/DataViews/BatteryView.mc

```diff
@@ -1,93 +1,101 @@
+        if (batteryBarHorizontal == true) //draw horizontal battery bar
         {
             if (battery > 99){
-                dc.fillRectangle(locX + barLength/2 - barThickness,  locY-8, barThickness, 8);
+                dc.fillRectangle(batteryBarLocX + batteryBarSize / 2 - batteryBarThickness,  batteryBarLocY - 8, batteryBarThickness, 8);
             }
             if (battery >= 75) {
-                dc.drawLine(locX*0.75 + barLength/2, locY, locX*0.75 + barLength/2, locY-5);
+                dc.drawLine(batteryBarLocX + batteryBarSize * 0.25 , batteryBarLocY, batteryBarLocX + batteryBarSize * 0.25, batteryBarLocY - 5);
             }
             if (battery >= 50) {
-                dc.drawLine(locX*0.5 + barLength/2, locY, locX*0.5 + barLength/2, locY-8);
+                dc.drawLine(batteryBarLocX , batteryBarLocY, batteryBarLocX, batteryBarLocY - 8);
             }
             if (battery >= 25) {
-                dc.drawLine(locX * 0.25 + barLength/2, locY, locX * 0.25 + barLength/2, locY-5);
+                dc.drawLine(batteryBarLocX - batteryBarSize * 0.25 , batteryBarLocY, batteryBarLocX - batteryBarSize * 0.25, batteryBarLocY - 5);
             }
-            dc.fillRectangle(locX - barLength/2, locY - 8,  barThickness, 8);
-            dc.fillRectangle(locX - barLength/2, locY, barLength / 100.0f * battery, barThickness);
+            dc.fillRectangle(batteryBarLocX - batteryBarSize / 2, batteryBarLocY - 8,  batteryBarThickness, 8);   //0% mark
+            dc.fillRectangle(batteryBarLocX - batteryBarSize / 2, batteryBarLocY, batteryPercentageBar, batteryBarThickness);
 
-
-        } else {
-            dc.fillRectangle(locX - 5, locY + barLength/2, 5, 1);
-            dc.fillRectangle(locX, locY + barLength/2, barThickness, barLength / 100.0f * battery);
-        }
+        } else {     // draw vertical battery bar
+            if (battery > 99){
+                dc.fillRectangle(batteryBarLocX - 8,  batteryBarLocY , 8, batteryBarThickness);
+            }
+            if (battery >= 75) {
+                dc.drawLine(batteryBarLocX - 5, batteryBarLocY  + batteryBarSize * 0.25, batteryBarLocX,  batteryBarLocY+ batteryBarSize * 0.25);
+            }
+            if (battery >= 50) {
+                dc.drawLine(batteryBarLocX - 8, batteryBarLocY  + batteryBarSize * 0.5, batteryBarLocX,  batteryBarLocY+ batteryBarSize * 0.5);
+            }
+            if (battery >= 25) {
+                dc.drawLine(batteryBarLocX - 5, batteryBarLocY  + batteryBarSize * 0.75, batteryBarLocX,  batteryBarLocY+ batteryBarSize * 0.75);
+            }
+            dc.fillRectangle(batteryBarLocX - 8, batteryBarLocY + batteryBarSize - batteryBarThickness, 8, batteryBarThickness);     //0% mark
+            dc.fillRectangle(batteryBarLocX,  batteryBarLocY + batteryPercentageOffset , batteryBarThickness, batteryPercentageBar);
         }
     }
 
 
     function drawBatteryBar(dc)
     {
-        var sysStats = System.getSystemStats();
```

Comment:
since the documentation also uses Sys, I would stay with it. I know this is not ideal, but I think this is the standard in the Connect IQ SDK community